static/app/components/group/externalIssuesList.tsx

```diff
@@ -57,8 +57,8 @@ class ExternalIssueList extends AsyncComponent<Props, State> {
     });
   }
 
-  UNSAFE_componentWillMount() {
-    super.UNSAFE_componentWillMount();
+  componentDidMount() {
+    super.componentDidMount?.();
```

Comment:
Super is `AsyncComponent`, which has a componentWillMount that does stuff. 
``` ts
    this.api = new Client();
    this.fetchData();

    if (this.reloadOnVisible) {
      document.addEventListener('visibilitychange', this.visibilityReloader);
    }
```

Reply:
yeah I moved some of this around too in there, it's also failing tests so, this one will prob change